### ISSUE TITLE
NTR: THCA high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8836]
New term request THCA high in urine (elthat_101118023615): Trihydrocholestanoic acid

**Comment by pnrobinson** [comment]
@MickeySegal

is this tetrahydrocannabinolic acid?

**Comment by MickeySegal** [comment]
As in https://github.com/obophenotype/human-phenotype-ontology/issues/8837#issuecomment-1627467217 this appears to be a typo now corrected in our source https://www.ncbi.nlm.nih.gov/books/NBK1448/ to "trihydroxycholestanoic".  

Not that this same typo applies to our finding now corrected to "THCA (trihydroxycholestanoic acid) high in blood" (segal_070920163940).

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000393
- label: Elevated urine trihydroxycholestanoic acid level
- parents: [Abnormal urine carbohydrate level (HP:0031979)](https://hpo.jax.org/app/browse/term/HP:0031979)
- pmid: 
- def: Presence of trihydroxycholestanoic acid in the urine. Trihydroxycholestanoic acid is an intermediate in the biosynthesis of cholic acid. Elevated plasma levels of trihydroxycholestanoic acid have been found in patients with Zellweger syndrome.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8836
- synonyms: Elevated urine THCA level(NONE)|Elevated urine trihydroxycoprostanic acid level(NONE)
